### DISCUSSION TITLE
Firefly Fails with MySQL 8.4

### DISCUSSION THREAD

**Discussion by lucasltt** [firefly-iii/firefly-iii#8859] [General]
Hello,

I would just like to share that Firefly 6.1.15 is failing with MySQL 8.4, released 4 days ago on docker hub.

#46 /var/www/html/public/index.php(73): Illuminate\\Foundation\\Http\\Kernel->handle()
#47 {main}
[previous exception] [object] (PDOException(code: 1524): SQLSTATE[HY000] [1524] Plugin 'mysql_native_password' is not loaded at /var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connectors/Connector.php:65)

Reverting back to MySql 8.3 made everything working again.

Hope this helps someone.

**Comment by JC5** [comment]
Loading the plug-in should fix that. :)

**Reply by lucasltt** [reply]
How should I do it? I'm using Firefly on Docker

**Reply by JC5** [reply]
I don't know, I use mariadb myself.
